src/utils/apiCalls.js

```diff
@@ -42,9 +42,23 @@ const getMessagesAfterId = async (channelId, limit, afterId, addFirstMessage = f
     return messages;
 }
 
+//returns the id of the message at the final index
+const getNumberMessages = async(channel, numberToSave, id) =>{
+    let messages;
+
+    //if id begin there
+    if (id) 
+        messages = await channel.messages.fetch({ cache: false, limit: numberToSave, before: id });
+    else
+        messages = await channel.messages.fetch({ cache: false, limit: numberToSave});
+            
+    return messages;
```

Comment:
Alternative I generally prefer in my own style:
```js
return id
    ? channel.messages.fetch({cache: false, limit: numberToSave, before: id})
    : channel.messages.fetch({cache: false, limit: numberToSave});
```
You also don't really need async/await here, as you're just returning the result of this `.fetch()`, which is itself a promise. So my suggestion is equivalent to your code.